### PR TITLE
Move to Capacitor 3 plugin layout

### PR DESCRIPTION
**Pull request by mahnuh**
We updated the plugin to use the current Capacitor 3 layout and also optimized the typings.

### CODE REVIEW

File: ios/Plugin/CapacitorHealthkit.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+@objc public class CapacitorHealthkit: NSObject {
```
Comment (arnauddrain) [inline ios/Plugin/CapacitorHealthkit.swift]: What is the purpose of this file?

**Review by arnauddrain — COMMENTED** [review]
Hello @mahnuh, thank you so much for your PR!
I just have a question about one of the new file but the rest looks good to me, I'll try to add a GitHub Action that check SwiftLint after merging the PR 🙂

File: ios/Plugin/CapacitorHealthkit.swift (hunk 1 above)
Comment (alemf) [reply ios/Plugin/CapacitorHealthkit.swift]: Hi,
This file is autogenerated on starting a new Capacitor 3 plugin. But the Plugin works also without it. So I removed it now.

**Comment by alemf** [comment]
Hi,
Do you think you will have time and energy to still merge this? :)
Otherwise we are thinking to publish our own fork.

Greetings!

**Comment by arnauddrain** [comment]
> Hi, Do you think you will have time and energy to still merge this? :) Otherwise we are thinking to publish our own fork.
> 
> Greetings!

Hello @alemf ,  I don't think I will have the time to maintain properly this extension, as I am not working on this subject anymore.
I will be glad to give you access to the npm package so you can keep the name. And mark this repo as deprecated with a link to your fork.
When you're ready, can you give me your npm username and a link to your fork?

**Comment by mahnuh** [comment]
Hi @arnauddrain would you drop me a line on Twitter so we can chat about how we could develop this plugin further?

**Pull request merged**